Makefile

```diff
@@ -31,7 +31,27 @@ ifeq ($(GOOS),windows)
 	IS_EXE := .exe
 endif
 MINISHIFT_BINARY ?= $(GOPATH)/bin/minishift$(IS_EXE)
-GODOG_OPTS ?= ""
+ifeq ($(filter-out integration,$(MAKECMDGOALS)),all)
+    GODOG_OPTS ?= -tags basic,cmd-openshift,command,coolstore,flags,openshift-version,proxy -format pretty
+endif
+ifeq ($(findstring .feature,$(filter-out integration,$(MAKECMDGOALS))), .feature)
```

Comment:
Filter out the goal ```integration``` and compare with the arguments if is there any feature file is passed to it
(findstring .feature,make integration ~/basic.feature, .feature) -> ~/basic.feature